spec/controllers/reports/regions_controller_spec.rb

```diff
@@ -59,6 +58,17 @@
       expect(response).to be_successful
     end
 
+    it "renders successfully for an organization when feature is enabled" do
+      admin = create(:admin, :viewer_reports_only, :with_access, resource: organization)
+      allow(admin).to receive(:feature_enabled?).with(:organization_reports).and_return(true)
+
+      sign_in(admin.email_authentication)
+      get :show, params: {id: organization.slug, report_scope: "organization"}
+
+      expect(response).to be_successful
+      expect(response.body).to include(organization.name)
+    end
+
```

Comment:
Some issues with the specs for this:

* This spec always passes. I changed the feature flag mock to return false, and the spec still passed. I deleted the `allow` line and it still passed.
* The reason it always passes is because we `show` the organization reports page in this, and the organization name shows up in the title of the page. And there is only an assertion on the presence of the organization name anywhere on the page, which is too broad and most likely to lead to false positives.
* There's no negative spec, ie, the link should not show up if the flag is disabled.

**suggestion**: These are the assertions we need at a minimum. And they should be a spec on the view template, not on the request, ie, like [this](https://github.com/simpledotorg/simple-server/blob/22eb0a44c9fbe9f7b5aeeb14c542be167c796260/spec/views/reports/regions/_diabetes_footnotes.html.erb_spec.rb) spec in the `/views` folder.

Make sure we are asserting on all of the following:
* When the flag is enabled and we are on the org-level report, the name should show up in the breadcrumb, and should not be a hyperlink.
* When the flag is enabled and we are on any report that's below the org-level (eg, state or facility) the org report should be in the breadcrumb and also should be a hyperlink that correctly navigates us to the org-level report.
* When the flag is disabled, the org name should NOT appear as a breadcrumb item. (test the negative case)

Reply:
@tfidfwastaken there is no specs for app/views/reports/regions/_header.html.erb ,
So i need to write the specs from initial stage,this will increase the scope of the ticket.

Reply:
Good, exhaustive specs need to be accompanied with every change we make to the code.

It should always be considered as part of the scope, and we need to consider the effort to add them in all our estimates from hereon.